gwtlib/github.py

```
@@ -0,0 +1,35 @@
+def get_pr_state(branch_name: str) -> Optional[Tuple[str, bool]]:
+    """Check the PR state for a branch using GitHub CLI.
+
+    Returns:
+        Tuple of (state, is_merged) where state is 'OPEN', 'CLOSED', or 'MERGED',
+        or None if no PR exists or gh CLI is not available.
+
+    Note: This uses the gh CLI and works in any directory with a GitHub remote.
+    """
+    try:
+        # Use gh pr view to get PR info for this branch
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)